public/app/features/datasources/state/buildCategories.ts

```diff
@@ -244,8 +244,9 @@ function getPhantomPlugin(options: GetPhantomPluginOptions): DataSourcePluginMet
       author: { name: 'Grafana Labs' },
       links: [
         {
-          url: config.pluginCatalogURL + options.id,
+          url: '/plugins/' + options.id,
```

Comment:
I am not quite sure if there is a better / stable way to import this route from somewhere as obviously if it changes this will break

Reply:
I am afraid we don't have these extracted to a const yet (they are defined [here as well](https://github.com/grafana/grafana/blob/207b2993b2e6f5c5ae6c42ad84ca740020f8d9a0/public/app/features/plugins/admin/routes.tsx#L11)). 

Probably a good candidate for refactoring in a separate PR!